### DISCUSSION TITLE
Winrm-ps-script

### DISCUSSION THREAD

**Discussion by mohit-singhal14** [StackStorm/st2#5712] [Documentation] [answered]
Hello
I am using runner-type as 'winrm-ps-script' to execute powershell script , however powershell script is executing but Now I want to pass parameters to powershell script.

I tried using env but not working.

Any suggestion will be appreciated.

Thanks . 

if you need more information about this query please comment on this

**Comment by amanda11** [comment, marked as answer]
The env sets environment variables, so your powershell script would need to read the values as environment variables rather than arguments.

Parameters to the script should be specified in the action's parameters:  see https://docs.stackstorm.com/actions.html#update-argument-parsing-in-the-script. Although the example is for a bash script, the same method should be able to use to send parameters for winrm-ps-script as well.
